### PR TITLE
REDCORE-476 [tests] structure api test for contacts

### PR DESCRIPTION
**Pull request by javigomez**
Do not merge, this is working progress. Depends on REDCORE-475

### CODE REVIEW

**Comment by javigomez** [comment]
Keeping in Work in progress. Structure tests now works but SOAP tests needs to be updated.

screen shot 2016-05-05 at 14 15 33

**Comment by nhungredweb** [comment]
@PhamHanh96 can you please help us fix the conflict?

File: tests/composer.json [hunk 1]
```diff
@@ -8,12 +8,12 @@
...
         "squizlabs/php_codesniffer": "1.*",
         "phpmd/phpmd": "*",
-        "codeception/codeception": "~2.1",
+        "codeception/codeception": "~2.2@dev",
         "phing/phing": "2.9.*@dev",
         "joomla-projects/joomla-browser": "v3.4.8.2",
         "codegyre/robo": "~0.6",
         "redcomponent/robo": "dev-master",
-        "redcomponent/teststatistics": "dev-master",
+        "redcomponent/teststatistics": "2.2",
```
Comment (puneet0191) [inline tests/composer.json]: as far as I know, this is outdated and doesn't work with the latest version of Codeception, please check it again @PhamHanh96 @nhungredweb

Comment (PhamHanh96) [reply tests/composer.json]: Thanks @puneet0191 I will check and fix it now. Thanks so much

**Pull request merged**